packages/graphql-config-utilities/src/polyfills.ts

```diff
@@ -0,0 +1,16 @@
+// config.ts uses Array.flatmap
+import 'core-js/modules/es7.array.flat-map';
```

Comment:
this seems like the least invasive way to pull in individual functions, couldn't find actual docs on this though...

Reply:
Is it our own code that uses flatMap? If so, can we just avoid it?

Reply:
yea i can probably do that